commit b8278009062c2a71a58f43f0feedfda61561e565 (HEAD -> master)
Author: Mike Zeller <mike@mikezeller.net>
Date:   2019-06-20T17:09:38-04:00 (4 months ago)
    
    TOOLS-2278 Exclude deps in rust-illumos-priv
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Cargo.toml b/Cargo.toml
index e2a28b5..40a5917 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -12,5 +12,6 @@ categories = [
     "os",
     "os::unix-apis"
 ]
+exclude = ["deps/*"]
 
 [dependencies]
